container/state.go

```diff
@@ -256,14 +255,14 @@ func (s *State) IsRestarting() bool {
 }
 
 // SetRemovalInProgress sets the container state as being removed.
```

Comment:
Can we update the doc to specify that `true` is returned if the container is already marked for removal `false` otherwise?